### PR TITLE
Migrating fun and misc commands and remove Prefix system + Introduced timeout command and numerous bug fixes.

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
# PR Title
### Checklist

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines](/CONTRIBUTING.md).
- [x] I agree with the [license](/LICENSE).

### Changes
- [x] Refactoring
- [x] Internal code
- [ ] External Code / Libraries
- [ ] Documentation
- [ ] Other: \_____

Closes Issue: -

## Description
I am migrating the fun and misc commands to slash command.

### CODE REVIEW

**Comment by RealYusufIsmail** [comment]
@SIMULATAN can you please review this pr after #181

**Comment by SIMULATAN** [comment]
sure

File: src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java:76]: shouldn't this be inverted?

File: src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java:125]: same as above

File: src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java:143]: Please extract the `entry.getValue().getValue()` as it gets called a good few times here

File: src/main/java/com/github/black0nion/blackonionbot/systems/antispoiler/AntiSpoilerSystem.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/systems/antispoiler/AntiSpoilerSystem.java:26]: This probably won't work because we also want to remove spoilers on non-interaction events

**Review by SIMULATAN — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java
Comment (RealYusufIsmail) [reply src/main/java/com/github/black0nion/blackonionbot/commands/bot/HelpCommand.java:76]: oh i see

**Comment by RealYusufIsmail** [comment]
This pr is blocked until #181 is merged

File: src/main/java/com/github/black0nion/blackonionbot/systems/ReactionRoleSystem.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/systems/ReactionRoleSystem.java:64]: Please translate this to "Unknown Emoji: '{}'"

File: src/main/java/com/github/black0nion/blackonionbot/systems/ReactionRoleSystem.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/systems/ReactionRoleSystem.java:40]: see below

File: src/main/resources/translations/English.json
Comment (SIMULATAN) [inline src/main/resources/translations/English.json:169]:
```suggestion
  "istimedout": "The member is already timed out.",
```

**Review by SIMULATAN — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/black0nion/blackonionbot/slashcommands/information/UserInfoCommand.java
Comment (SIMULATAN) [inline src/main/java/com/github/black0nion/blackonionbot/slashcommands/information/UserInfoCommand.java:30]: missing space

**Review by SIMULATAN — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
